modules/income_limits/app/controllers/income_limits/v1/income_limits_controller.rb

```diff
@@ -1,75 +1,125 @@
 # frozen_string_literal: true
 
-# Income limits api response data shape example:
-# {
-#   "county_name": "Some County, XX",
-#   "income_year": 2022,
-#   "limits": {
-#     "national_threshold": {
-#       "0_depedent": 99999,
-#       "1_dependent": 99999,
-#       "additional_dependents": 9999
-#     },
-#     "relaxation_threshold": {
-#       "0_dependent": 99999,
-#       "1_dependent": 99999,
-#       "additional_dependents": 9999
-#     },
-#     "housebound_threshold": {
-#       "0_dependent": 99999,
-#       "1_dependent": 99999,
-#       "additional_dependents": 9999
-#     },
-#     "aid_attendence_threshold": {
-#       "0_dependent": 99999,
-#       "1_dependent": 99999,
-#       "additional_dependents": 9999
-#     },
-#     "pension_threshold": {
-#       "0_dependent": 99999,
-#       "1_dependent": 99999,
-#       "additional_dependents": 9999
-#     },
-#     "gmt_threshold": 9999
-#   }
-# }
-
 module IncomeLimits
   module V1
     class IncomeLimitsController < ApplicationController
       skip_before_action :authenticate
 
       def index
-        example_data = {
-          county_name: 'Some County, XX',
-          income_year: 2022,
-          limits: {
-            national_threshold: { '0_depedent': 99_999, '1_dependent': 99_999, additional_dependents: 9999 },
-            relaxation_threshold: { '0_dependent': 99_999, '1_dependent': 99_999, additional_dependents: 9999 },
-            housebound_threshold: { '0_dependent': 99_999, '1_dependent': 99_999, additional_dependents: 9999 },
-            aid_attendence_threshold: { '0_dependent': 99_999, '1_dependent': 99_999, additional_dependents: 9999 },
-            pension_threshold: { '0_dependent': 99_999, '1_dependent': 99_999, additional_dependents: 9999 },
-            gmt_threshold: 9999
-          }
+        zip = params[:zip].to_i
```

Comment:
I always thought zip codes needed to stay as strings because New England zip codes start with zero and that zero will get stripped with `to_i`.
```
irb(main):002:0> "04962".to_i
=> 4962
```

Reply:
@jtmst here is some more info I found: https://stackoverflow.com/questions/893454/is-it-a-good-idea-to-use-an-integer-column-for-storing-us-zip-codes-in-a-databas

Reply:
The data has been updated and the zip codes are now strings